rbac/ledger_sync/inbound/listener.py

```diff
@@ -113,7 +113,9 @@ def listener():
 
         LOGGER.info("Reading queued Sawtooth transactions")
         while True:
-            feed = database.run_query(database.get_table("inbound_queue"))
+            feed = database.run_query(
+                database.get_table("inbound_queue").order_by(index=r.asc("timestamp"))
```

Comment:
This is using the old methodology of database.  Can you fix once Database develop sync is merged?

Reply:
Will do!